coderd/prometheusmetrics/aggregator.go

```diff
@@ -146,13 +147,40 @@ func (ma *MetricsAggregator) Run(ctx context.Context) func() {
 
 				output := make([]prometheus.Metric, 0, len(ma.queue))
 				for _, m := range ma.queue {
-					desc := prometheus.NewDesc(m.Name, metricHelpForAgent, agentMetricsLabels, nil)
+					labels := make([]string, len(agentMetricsLabels)+len(m.Labels))
+					var i int
```

Comment:
nit: rename this to be more descriptive; it appears to serve an important function.

Reply:
I'm not sure if I know what you mean. Are you referring to the variable `i`? I can rename it to `labelIndex`, I guess?

Reply:
Yes, correct. `labelIndex` would be fine!